tests/integration/Germline_PairedEnd.json

```diff
@@ -1,4 +1,5 @@
 {
+"Germline.outputDir":"/home/rhpvorderman/PycharmProjects/germline-DNA/test-output",
```

Comment:
Is this a correct output directory?

Reply:
good catch. Don't know why it did this?

Reply:
Whoops, seems I run a `sed` command in the root directory.